app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js

```diff
@@ -22,6 +22,12 @@ var gas_amount = function(page_url) {
     gasLimitEstimate = 21000;
   } else if (page_url.indexOf('tip/receive') != -1) { // tip
     gasLimitEstimate = 21000;
+  } else if (page_url.indexOf('/fund') != -1) { // grant contribution
+    gasLimitEstimate = 318730;
```

Comment:
funding / creating a new subscription cost the same gas-wise?  that doesnt seem right